index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="body-wrapper">
+      <!-- nav section -->
+      <div class="mobile-nav">
+        <h1>Welcome</h1>
```

Comment:
- [ ] If you notice, most of the websites have a logo, and when you click on it you are directed to the homepage 🚶 . This implies that the logo is a link with `href`. This enables the users easily navigate to the homepage. Therefore, I suggest that we consider wrapping the logo in `a` tag ☺️ .
- [ ] You have done great by ensuring you opt to use `h1` HTML tag 👏 . However, in this section let's consider using `a` tag as suggested above. This is because to be [Semantically](https://www.w3schools.com/html/html5_semantic_elements.asp) correct, the page should have only one `h1` which is the main heading of the page. In addition, doing this increases the accessibility of the page 👍 .